app/requirements.in

```
@@ -16,3 +16,4 @@ grpcio==1.59.0
 protobuf==4.24.4
 dapr==1.11.0
 cloudevents==1.10.0
+aiohttp >= 3.9.0
```

```suggestion
aiohttp == 3.9.0
```
To align with our "freeze version" strategy

---

No need to recompile - still same version was selected

---

already updated